src/webui/www/private/views/preferences.html

```diff
@@ -1735,6 +1743,7 @@
                         $('resolvePeerCountries').setProperty('checked', pref.resolve_peer_countries);
                         // libtorrent section
                         $('asyncIOThreads').setProperty('value', pref.async_io_threads);
+                        $('stop_tracker_timeout').setProperty('value', pref.stop_tracker_timeout);
```

Comment:
After changing above to camelCase, `$('stop_tracker_timeout')` should also be changed to `$('stopTrackerTimeout')`